src/vmm/Cargo.toml

```diff
@@ -24,17 +24,22 @@ versionize_derive = "0.1.5"
 vm-allocator = "0.1.0"
 vm-superio = "0.7.0"
 vm-fdt = "0.2.0"
+timerfd = "1.2.0"
+rand = "0.8.5"
```

Comment:
nit: It seems to be arranged in dictionary order.

Reply:
Fixed.

Reply:
Created https://github.com/firecracker-microvm/firecracker/pull/3701 to cover this.